src/metrics/generated/proto/metricpb/custom_unmarshal.go

```diff
@@ -0,0 +1,432 @@
+// Copyright (c) 2021 Uber Technologies, Inc.
```

Comment:
Will this file need to be manually generated again if the fields change?

Reply:
yea